.github/workflows/backport-assistant.yml

```diff
@@ -41,3 +42,15 @@ jobs:
             -X POST \
             -d "{ \"body\": \"${github_message}\"}" \
             "https://api.github.com/repos/${GITHUB_REPOSITORY}/issues/${{ github.event.pull_request.number }}/comments"
+  backport-ent:
+    if: github.event.pull_request.merged && contains(join(github.event.pull_request.labels.*.name), 'backport/ent')
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.4.0
+    steps:
+      - name: Trigger backport for Enterprise
+        uses: peter-evans/repository-dispatch@ff45666b9427631e3450c54a1bcbee4d9ff4d7c0 # v3.0.0
```

Comment:
~ Not my area of expertise, but curious whether we should just use a raw curl like in https://github.com/hashicorp/consul/blob/main/.github/workflows/ce-merge-trigger.yml given the amount of config is about the same - the risk of leaking a privileged GH token seems relatively high in the case of a supply-chain attack in future versions of this action

Reply:
The action should be OK to use since it's in the tsccr database. Security manages it, so I think it's OK to trust it.